### DISCUSSION TITLE
Filtering option for (case) tasks assigned to me

### DISCUSSION THREAD

**Discussion by ar3diu** [reflexsoar/reflex-api#184] [Q&A]
Hi,

How can a analyst see the (case) tasks assigned to him? :thinking: 
I may be missing something, but I still haven't been able to find that filter option.

**Comment by n3tsurge** [comment]
For case tasks it's not possible yet, it's coming shortly.

Brian Carroll
________________________________
From: Andrei R ***@***.***>
Sent: Friday, May 20, 2022 7:39:39 AM
To: reflexsoar/reflex-api ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [reflexsoar/reflex-api] Filtering option for (case) tasks assigned to me (Discussion #184)


Hi folks,

How can a analyst see the (case) tasks assigned to him? 🤔
I may be missing something, but I still haven't been able to find that filter option.

**Comment by ar3diu** [comment]
Thanks for the quick response, Brian! 
And thank you anyway for open-sourcing this project. You're doing a great job! :pray:

**Reply by n3tsurge** [reply]
Appreciate the kind words.  Wish I could get some of this Quality of Life stuff out faster.  Detections engine is our main priority right now so that you have end to end alert life cycle in one spot.  After that I expect some QoL stuff and case management improvements to follow.

**Reply by n3tsurge** [reply]
I have opened an official issue to track the work on this #185

**Reply by ar3diu** [reply]
I'm really curious about the new features. I've noticed the Playbooks menu option and I guess that too will come soon.
Also, mail settings... can we set email notifications at the moment, let's say for all new cases?

**Reply by n3tsurge** [reply]
Notifications will be coming shortly after Detections when we rework some of the Case Management stuff allowing for @mentions and status change notifications, etc.  Playbooks are a little further out while we hammer home a pluggable architecture that makes it easy to add third-party integrations and actions that can then be integrated in to Playbooks.  Some of the menu items you are seeing are placeholders from early testing that I guess I didn't remove.

**Reply by ar3diu** [reply]
Sounds very good! Thanks for all these news!
So far, alert de-duplication through signature fields has made my life so much easier.

**Reply by n3tsurge** [reply]
@anrsec if you haven't seen it yet we are tracking our Roadmap here https://github.com/orgs/reflexsoar/projects/4/views/1 so you can see what stuff is coming.  Our versioning scheme follows targeted release month so you'll know exactly what month we are targeting the features for.

**Reply by ar3diu** [reply]
Thanks, I hadn't seen it yet! Lots of goodies there :grinning:

**Comment by SMAPPER** [comment]
That's fantastic to read. We are glad it's helping you.
